### ISSUE TITLE
RTL8723BE

### ISSUE THREAD

**Issue by AhmedEsmat96** [lwfinger/rtlwifi_new#162]
my wifi version is RTL8723BE but after make and make install not working what should i do

**Comment by lwfinger** [comment]
A better description of the problem and any messages from the logs would be a start. You just gave me the equivalent of "My car won't start. What should I do?"

**Comment by AhmedEsmat96** [comment]
this after sudo make install
make -C /lib/modules/4.7.9-100.fc23.x86_64/build M=/home/ahmed/rtlwifi_new modules
make[1]: Entering directory '/usr/src/kernels/4.7.9-100.fc23.x86_64'
  Building modules, stage 2.
  MODPOST 14 modules
make[1]: Leaving directory '/usr/src/kernels/4.7.9-100.fc23.x86_64'
Install rtlwifi SUCCESS



but still not work

**Comment by lwfinger** [comment]
Did you load the module, or reboot? Merely adding the module to a running system does nothing unless you trigger a bus scan, but that is difficult to do.

Please run the following two commands and report the output.

sudo modprobe -rv rtl8723be
sudo modprobe -v rtl8723be

**Comment by AhmedEsmat96** [comment]
no thing work but when i enter this command 
rfkill list   it gave me  this 
0: acer-wireless: Wireless LAN
	Soft blocked: yes
	Hard blocked: no
1: phy0: Wireless LAN
	Soft blocked: no
	Hard blocked: no
2: hci0: Bluetooth
	Soft blocked: no
	Hard blocked: no

**Comment by lwfinger** [comment]
It has something to do with the Acer-specific code. As my laptop is made by Toshiba, I cannot help that part. Does the command 'sudo rfkill unblock all' help?

**Comment by AhmedEsmat96** [comment]
i doesn't help thank you for your help

**Comment by lwfinger** [comment]
Try the command 'sudo modprobe -rv acer-wmi'. Does that enable wifi? If it does, and your laptop is not made by Acer, then you can blacklist acer-wmi. If you have an Acer laptop, a lot of other things will probably break. You will need to contact the list at platform-driver-x86@vger.kernel.org to get help on debugging that routine.

**Comment by AhmedEsmat96** [comment]
my laptop is HP-envy x360

**Comment by AhmedEsmat96** [comment]
but what did this command do?

**Comment by AhmedEsmat96** [comment]
yes it work thank you

**Comment by AhmedEsmat96** [comment]
but i want to know what this command do

**Comment by lwfinger** [comment]
Which command? I gave you several. The modprobe command removed a driver.

**Comment by lwfinger** [comment]
A completely new version of the drivers has just been pushed. It fixes a lot of things. As a result, I am closing all open issues. If you get a new one, please create a new issue.

Note: The new driver fixes the problems with power save. It is recommended that you clear all ips, swlps, and fwlps options.
